src/fpm.f90

```diff
@@ -502,4 +504,45 @@ end subroutine compact_list
 
 end subroutine cmd_run
 
+subroutine delete_skip(unix)
+    !> delete directories in the build folder, skipping dependencies
+    logical, intent(in) :: unix
+    character(len=:), allocatable :: dir
+    type(string_t), allocatable :: files(:)
+    integer :: i
+    call list_files('build', files, .false.)
+    do i = 1, size(files)
+        if (is_dir(files(i)%s)) then
+            dir = files(i)%s
+            if (dir /= 'build/dependencies') call os_delete_dir(unix, dir)
```

Comment:
This test for `'build/dependencies'` isn't working on Windows, because of the different directory separator. I suggest using [`str_ends_with`](https://fortran-lang.github.io/fpm/module/fpm_strings.html#interface-str_ends_with) from `fpm_strings` to test for this condition. Perhaps:

```suggestion
            if (.not.str_ends_with(dir,'dependencies')) call os_delete_dir(unix, dir)
```